libc/src/__support/FPUtil/FEnvImpl.h

```diff
@@ -11,7 +11,6 @@
 
 #include "include/llvm-libc-macros/math-macros.h"
 #include "src/__support/macros/attributes.h" // LIBC_INLINE
-#include "src/__support/macros/config.h"     // LIBC_HAS_BUILTIN
```

Comment:
This change allows us to remove the dependency on `libc.src.__support.macros.config` from `fenv_impl` in libc/src/__support/FPUtil/CMakeLists.txt.

Reply:
Done.